### PR TITLE
  Bug 1183970 - Add a test for removing cells from top sites

### PR DESCRIPTION
**Pull request by bkmunar**
1. rebased from wesj's patch
2. landing in 1.1+ after no string freeze

### CODE REVIEW

File: UITests/TopSitesTests.swift [hunk 1]
```diff
@@ -58,6 +80,32 @@ class TopSitesTests: KIFTestCase {
...
+        // Open top sites
+        tester().tapViewWithAccessibilityIdentifier("url")
+        tester().tapViewWithAccessibilityLabel("Top sites")
+
+        // Verify the row exists and that the Remove Page button is hidden
+        let row = tester().waitForViewWithAccessibilityLabel("Page 1")
+        tester().waitForAbsenceOfViewWithAccessibilityLabel("Remove page")
+
+        // Long press the row and click the remove button
+        row.longPressAtPoint(CGPoint(x: 0,y: 0), duration: 1)
```
Comment (thebnich) [inline UITests/TopSitesTests.swift]: Nit: I think there's a `CGPointZero`

**Comment by thebnich** [comment]
This might not work well when run with other tests since history/top sites aren't cleared between each one. I guess that's a separate bug, though.

**Comment by thebnich** [comment]
@bkmunar, could you rebase this by removing the merge commit/"nitz" commit?

**Comment by bkmunar** [comment]
Fixed everything you asked for mah deer bnicholz, lemme know when to mergeth

**Pull request closed without merging**